src/groovy/org/pih/warehouse/requisition/RequisitionStatus.groovy

```diff
@@ -70,15 +70,18 @@ enum RequisitionStatus {
         [CREATED, EDITING, VERIFYING, PICKING, PICKED, CHECKING, ISSUED, CANCELED, PENDING, REQUESTED]
     }
 
+    // Default options for outbound list, without supporting request approval
     static listOutboundOptions() {
         [CREATED, EDITING, VERIFYING, PICKING, PICKED, CHECKING, ISSUED, CANCELED, PENDING, REQUESTED, DISPATCHED]
     }
 
+    // Options for outbounds when we are supporting request approval (Added approved)
     static listOutboundOptionsWhenSupportingRequestApproval() {
         [CREATED, EDITING, APPROVED, PICKING, PICKED, CHECKING, ISSUED, CANCELED, PENDING, REQUESTED, DISPATCHED]
     }
 
-    static listRequisitionOptions() {
+    // Options for request list when we are supporting request approval (Added approved and waiting for approval)
+    static listRequestOptionsWhenSupportingRequestApproval() {
```

Comment:
I'm stlil not sure about the naming for this one, but I don't have any suggestion so I'll leave it for others maybe

Reply:
Yeah this is something that we could think about a bit

Reply:
I assume we don't want to spend time thinking about naming this one? @awalkowiak ?